gapis/api/cmd_id_group.go

```diff
@@ -413,18 +432,63 @@ func (c *SubCmdRoot) Insert(base []uint64, r []uint64) {
 		}
 		sg.Insert(append(base, r[0]), r[1:])
 	} else {
-		if len(c.SubGroup.Spans) == 0 {
-			c.SubGroup.Spans = Spans{&CmdIDRange{CmdID(0), id + 1}}
+		// Add subcommand one by one to the sub groups of the SubCmdRoot
```

Comment:
OK

Reply:
please can we have better docs on this function.

Reply:
Fixed in #1064